packages/react-ui/src/widgets/RangeWidgetUI.js

```diff
@@ -138,35 +149,50 @@ function RangeWidgetUI({ data, min, max, limits, onSelectedRangeChange }) {
     onSelectedRangeChange && onSelectedRangeChange(newValues);
   };
 
+  const resetSlider = () => {
+    const newValues = [min, max];
+    setInputsValues(newValues);
+    setSliderValues([...newValues]);
```

Comment:
Do we need here destructuring?

Reply:
Probably not but it's used in other pieces of the component. I assumed that @padawannn found some problem and needed to use destructuring here:
https://github.com/CartoDB/carto-react/blob/8d552890186d2d34bfdac94aed4e32d6766d9217/packages/react-ui/src/widgets/RangeWidgetUI.js#L148